src/Sylius/Behat/Context/Api/Admin/AjaxContext.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Model\ProductInterface;
+use Symfony\Component\BrowserKit\AbstractBrowser;
+use Symfony\Component\BrowserKit\Cookie;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Webmozart\Assert\Assert;
+
+final class AjaxContext implements Context
+{
+    private AbstractBrowser $client;
+
+    private SessionInterface $session;
+
+    public function __construct(AbstractBrowser $client, SessionInterface $session)
+    {
+        $this->client = $client;
+        $this->session = $session;
+    }
+
+    /**
+     * @When I look for a variant with :phrase in descriptor within the :product product
+     */
+    public function iLookForVariantWithDescriptorWithinProduct($phrase, ProductInterface $product): void
```

Comment:
```suggestion
    public function iLookForVariantWithDescriptorWithinProduct(string $phrase, ProductInterface $product): void
```

Also in other methods the type is missing 😢

Reply:
I've only changed the name of this file without any changes in it and I would like to leave it as it is in this PR 😃